### ISSUE TITLE
Add note in button a11y section to use text-transform for capitalizing buttons

### ISSUE THREAD

**Issue by andrewseguin** [angular/components#17078]
#### Documentation Feedback

Material design buttons are always shown with all-capital letters in the spec: https://material.io/components/buttons/#anatomy

It should be noted to users that this should be done with `text-transform` rather than actually writing out the text in all-capital letters. Otherwise screen-readers will spell out each letter as if it is an acronym

**Affected documentation page:** https://material.angular.io/components/button/examples

**Comment by crisbeto** [comment]
We have this note about it already: https://material.angular.io/components/button/overview#capitalization

**Comment by crisbeto** [comment]
Closing since we have a section in docs about this already.
